.atmos.d/dev.yaml

```
@@ -0,0 +1,96 @@
+            echo "  make build       - Build Atmos"
+            echo "  make test        - Run tests"
+            echo "  make lint        - Lint code"
+            echo "  atmos dev check  - Run pre-commit checks"
+            echo "  atmos dev help   - Show all dev commands"
+      
+      # Pre-commit workflow
+      - name: check
+        description: Run pre-commit checks on staged files
+        steps:
+          - pre-commit run
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)